Assets/Scripts/SetGravity.cs

```diff
@@ -1,13 +0,0 @@
-using System.Collections;
```

Comment:
I think we do want this one actually, I made this so you can adjust the gravity constant